clab/clab_test.go

```diff
@@ -220,37 +167,6 @@ func getNodeMap(mockCtrl *gomock.Controller) map[string]nodes.Node {
 	return nodeMap
 }
 
-func Test_createWaitForDependency(t *testing.T) {
-	mockCtrl := gomock.NewController(t)
-	defer mockCtrl.Finish()
```

Comment:
we have robot tests for it, yes, but I am curious why you removed this?

Reply:
see above